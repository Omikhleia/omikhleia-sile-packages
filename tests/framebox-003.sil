\begin{document}
\script[src=packages/framebox]
\script[src=packages/struts]
\nofolios
\neverindent
% Test case: \bracebox basics

A default \bracebox{\strut{}left-braced} box

\medskip

A default \bracebox[side=right]{\strut{}right-braced} box

\medskip

A default \bracebox[side=both]{\strut{}braced} box

\medskip

No thickness \bracebox[bracethickness=0, side=both]{\strut{}braced} box

\medskip

Not so curvy \bracebox[side=both, curvyness=0.5]{\strut{}braced} box

\medskip

More curvy \bracebox[side=both, curvyness=0.725]{\strut{}braced} box

\medskip

Blue \bracebox[side=both, bracecolor=blue]{\strut{}braced} box

\medskip

Larger \bracebox[side=both, bracewidth=0.5em]{\strut{}braced} box

\medskip

Heavy \bracebox[side=both, strokewidth=0.1em]{\strut{}braced} box

\end{document}
