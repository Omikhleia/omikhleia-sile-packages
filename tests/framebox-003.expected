Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	50.2488
Set font 	Gentium Plus;10;400;;normal;;LTR
T	36 w=5.9521	(A)
Mx 	38.4056
T	71 72 73 68 88 79 87 w=28.9697	(default)
Draw SVG	50.2488	50.0684	10.5957	1.0000	2.5 2.44140625 m 1 2.44140625 1.5 5.09033203125 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 5.09033203125 y 1 2.44140625 2.5 2.44140625 y 2.5 13.037109375 m 1 13.037109375 1.5 10.38818359375 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 10.38818359375 y 1 13.037109375 2.5 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B
Mx 	75.0650
T	79 72 73 87 w=13.8818	(left)
Mx 	88.9468
T	16 w=3.3691	(-)
Mx 	92.3159
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	122.8230
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	68.2488
T	36 w=5.9521	(A)
Mx 	38.4049
T	71 72 73 68 88 79 87 w=28.9697	(default)
Draw SVG	68.2488	56.6943	10.5957	1.0000	54.1943359375 2.44140625 m 55.6943359375 2.44140625 55.1943359375 5.09033203125 v 54.6943359375 7.7392578125 56.6943359375 7.7392578125 v 53.6943359375 7.7392578125 54.6943359375 5.09033203125 y 55.6943359375 2.44140625 54.1943359375 2.44140625 y 54.1943359375 13.037109375 m 55.6943359375 13.037109375 55.1943359375 10.38818359375 v 54.6943359375 7.7392578125 56.6943359375 7.7392578125 v 53.6943359375 7.7392578125 54.6943359375 10.38818359375 y 55.6943359375 13.037109375 54.1943359375 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B
Mx 	70.0635
T	85 76 74 75 87 w=20.5078	(right)
Mx 	90.5713
T	16 w=3.3691	(-)
Mx 	93.9405
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	129.4468
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	86.2488
T	36 w=5.9521	(A)
Mx 	38.4069
T	71 72 73 68 88 79 87 w=28.9697	(default)
Draw SVG	86.2488	37.8174	10.5957	1.0000	2.5 2.44140625 m 1 2.44140625 1.5 5.09033203125 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 5.09033203125 y 1 2.44140625 2.5 2.44140625 y 2.5 13.037109375 m 1 13.037109375 1.5 10.38818359375 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 10.38818359375 y 1 13.037109375 2.5 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B 35.3173828125 2.44140625 m 36.8173828125 2.44140625 36.3173828125 5.09033203125 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 34.8173828125 7.7392578125 35.8173828125 5.09033203125 y 36.8173828125 2.44140625 35.3173828125 2.44140625 y 35.3173828125 13.037109375 m 36.8173828125 13.037109375 36.3173828125 10.38818359375 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 34.8173828125 7.7392578125 35.8173828125 10.38818359375 y 36.8173828125 13.037109375 35.3173828125 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B
Mx 	75.0677
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	110.5760
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	104.2488
T	49 82 w=11.5186	(No)
Mx 	43.9716
T	87 75 76 70 78 81 72 86 86 w=38.7793	(thickness)
Draw SVG	104.2488	37.8174	10.5957	1.0000	2.5 2.44140625 m 1 2.44140625 1.5 5.09033203125 v 2 7.7392578125 0 7.7392578125 v 2 7.7392578125 1.5 5.09033203125 y 1 2.44140625 2.5 2.44140625 y 2.5 13.037109375 m 1 13.037109375 1.5 10.38818359375 v 2 7.7392578125 0 7.7392578125 v 2 7.7392578125 1.5 10.38818359375 y 1 13.037109375 2.5 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B 35.3173828125 2.44140625 m 36.8173828125 2.44140625 36.3173828125 5.09033203125 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 35.8173828125 7.7392578125 36.3173828125 5.09033203125 y 36.8173828125 2.44140625 35.3173828125 2.44140625 y 35.3173828125 13.037109375 m 36.8173828125 13.037109375 36.3173828125 10.38818359375 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 35.8173828125 7.7392578125 36.3173828125 10.38818359375 y 36.8173828125 13.037109375 35.3173828125 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B
Mx 	90.4403
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	125.9469
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	122.2488
T	49 82 87 w=14.9609	(Not)
Mx 	47.4141
T	86 82 w=8.8916	(so)
Mx 	58.9950
T	70 88 85 89 92 w=23.4912	(curvy)
Draw SVG	122.2488	37.8174	10.5957	1.0000	2.5 2.44140625 m 1.25 2.44140625 1.25 5.09033203125 v 1.25 7.7392578125 0 7.7392578125 v 2.25 7.7392578125 1.75 5.09033203125 y 1.25 2.44140625 2.5 2.44140625 y 2.5 13.037109375 m 1.25 13.037109375 1.25 10.38818359375 v 1.25 7.7392578125 0 7.7392578125 v 2.25 7.7392578125 1.75 10.38818359375 y 1.25 13.037109375 2.5 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B 35.3173828125 2.44140625 m 36.5673828125 2.44140625 36.5673828125 5.09033203125 v 36.5673828125 7.7392578125 37.8173828125 7.7392578125 v 35.5673828125 7.7392578125 36.0673828125 5.09033203125 y 36.5673828125 2.44140625 35.3173828125 2.44140625 y 35.3173828125 13.037109375 m 36.5673828125 13.037109375 36.5673828125 10.38818359375 v 36.5673828125 7.7392578125 37.8173828125 7.7392578125 v 35.5673828125 7.7392578125 36.0673828125 10.38818359375 y 36.5673828125 13.037109375 35.3173828125 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B
Mx 	90.1755
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	125.6823
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	140.2488
T	48 82 85 72 w=21.6699	(More)
Mx 	54.1236
T	70 88 85 89 92 w=23.4912	(curvy)
Draw SVG	140.2488	37.8174	10.5957	1.0000	2.5 2.44140625 m 0.6875 2.44140625 1.8125 5.09033203125 v 2.9375 7.7392578125 0 7.7392578125 v 3.9375 7.7392578125 2.3125 5.09033203125 y 0.6875 2.44140625 2.5 2.44140625 y 2.5 13.037109375 m 0.6875 13.037109375 1.8125 10.38818359375 v 2.9375 7.7392578125 0 7.7392578125 v 3.9375 7.7392578125 2.3125 10.38818359375 y 0.6875 13.037109375 2.5 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B 35.3173828125 2.44140625 m 37.1298828125 2.44140625 36.0048828125 5.09033203125 v 34.8798828125 7.7392578125 37.8173828125 7.7392578125 v 33.8798828125 7.7392578125 35.5048828125 5.09033203125 y 37.1298828125 2.44140625 35.3173828125 2.44140625 y 35.3173828125 13.037109375 m 37.1298828125 13.037109375 36.0048828125 10.38818359375 v 34.8798828125 7.7392578125 37.8173828125 7.7392578125 v 33.8798828125 7.7392578125 35.5048828125 10.38818359375 y 37.1298828125 13.037109375 35.3173828125 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B
Mx 	85.3047
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	120.8119
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	158.2488
T	37 79 88 72 w=18.1250	(Blue)
Draw SVG	158.2488	37.8174	10.5957	1.0000	2.5 2.44140625 m 1 2.44140625 1.5 5.09033203125 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 5.09033203125 y 1 2.44140625 2.5 2.44140625 y 2.5 13.037109375 m 1 13.037109375 1.5 10.38818359375 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 10.38818359375 y 1 13.037109375 2.5 13.037109375 y 1 J 1 j 0 0 1 RG 0 0 1 rg 0.33 w B 35.3173828125 2.44140625 m 36.8173828125 2.44140625 36.3173828125 5.09033203125 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 34.8173828125 7.7392578125 35.8173828125 5.09033203125 y 36.8173828125 2.44140625 35.3173828125 2.44140625 y 35.3173828125 13.037109375 m 36.8173828125 13.037109375 36.3173828125 10.38818359375 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 34.8173828125 7.7392578125 35.8173828125 10.38818359375 y 36.8173828125 13.037109375 35.3173828125 13.037109375 y 1 J 1 j 0 0 1 RG 0 0 1 rg 0.33 w B
Mx 	55.5819
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	91.0925
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	176.2488
T	47 68 85 74 72 85 w=26.7676	(Larger)
Draw SVG	176.2488	42.8174	10.5957	1.0000	5 2.44140625 m 2 2.44140625 3 5.09033203125 v 4 7.7392578125 0 7.7392578125 v 5 7.7392578125 3.5 5.09033203125 y 2 2.44140625 5 2.44140625 y 5 13.037109375 m 2 13.037109375 3 10.38818359375 v 4 7.7392578125 0 7.7392578125 v 5 7.7392578125 3.5 10.38818359375 y 2 13.037109375 5 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B 37.8173828125 2.44140625 m 40.8173828125 2.44140625 39.8173828125 5.09033203125 v 38.8173828125 7.7392578125 42.8173828125 7.7392578125 v 37.8173828125 7.7392578125 39.3173828125 5.09033203125 y 40.8173828125 2.44140625 37.8173828125 2.44140625 y 37.8173828125 13.037109375 m 40.8173828125 13.037109375 39.8173828125 10.38818359375 v 38.8173828125 7.7392578125 42.8173828125 7.7392578125 v 37.8173828125 7.7392578125 39.3173828125 10.38818359375 y 40.8173828125 13.037109375 37.8173828125 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 0.33 w B
Mx 	66.7230
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	104.7320
T	69 82 91 w=15.1367	(box)
Mx 	29.7638
My 	194.2488
T	43 72 68 89 92 w=25.6104	(Heavy)
Draw SVG	194.2488	37.8174	10.5957	1.0000	2.5 2.44140625 m 1 2.44140625 1.5 5.09033203125 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 5.09033203125 y 1 2.44140625 2.5 2.44140625 y 2.5 13.037109375 m 1 13.037109375 1.5 10.38818359375 v 2 7.7392578125 0 7.7392578125 v 3 7.7392578125 2 10.38818359375 y 1 13.037109375 2.5 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 1 w B 35.3173828125 2.44140625 m 36.8173828125 2.44140625 36.3173828125 5.09033203125 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 34.8173828125 7.7392578125 35.8173828125 5.09033203125 y 36.8173828125 2.44140625 35.3173828125 2.44140625 y 35.3173828125 13.037109375 m 36.8173828125 13.037109375 36.3173828125 10.38818359375 v 35.8173828125 7.7392578125 37.8173828125 7.7392578125 v 34.8173828125 7.7392578125 35.8173828125 10.38818359375 y 36.8173828125 13.037109375 35.3173828125 13.037109375 y 1 J 1 j 0 0 0 RG 0 0 0 rg 1 w B
Mx 	63.0665
T	69 85 68 70 72 71 w=27.8174	(braced)
Mx 	98.5762
T	69 82 91 w=15.1367	(box)
End page
Finish
